pandas/conftest.py

```diff
@@ -978,6 +978,7 @@ def __len__(self):
 
 @pytest.fixture(params=indices_dict.keys())
 def indices(request):
+    """ Fixture for all kinds of indices. """
```

Comment:
Can this specify that the indices are "simple" not likely to cover corner cases, e.g. no names, no NaTs, no values near implementation bounds...